primitives/state-machine/src/overlayed_changes/changeset.rs

```diff
@@ -69,7 +69,6 @@ struct InnerValue<V> {
 	/// Current value. None if value has been deleted.
 	value: V,
 	/// The set of extrinsic indices where the values has been changed.
-	/// Is filled only if runtime has announced changes trie support.
 	extrinsics: Extrinsics,
```

Comment:
Should we just remove this `Extrinsics` field (I remember thinking of some optimization to it, but not having it is a better optimization).

Reply:
I thinks this is also involved in supporting transactions in the runtime. I.e. to revert storage changes when extrinsic fails. In any case I'd rather do complex logic changes and optimisations in a separate PR.

Reply:
+1 for separate, IIRC transaction stuff just merge extrinsic when commiting (it is just additional computation).